surefire-its/src/test/resources/junit5-modulepath/src/main/java/com/app/Main.java

```diff
@@ -0,0 +1,34 @@
+package com.app;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.joda.time.DateTime;
+
+public class Main
+{
+    public static void main( String... args )
+    {
+        System.out.println( "module path => " + System.getProperty( "jdk.module.path" ) );
```

Comment:
So, these lines are used as a debug info?
You can maybe verify the classpath in one line, and modulapath in the second line via combining Java Hamcrest matchers.
Example `validator.assertThatLogLine( containsString( "Running jiras.surefire1082.Jira1082Test" ), is( 1 ) );`
Example with matching one line: `allOf( containsString( "jar1" ), containsString( "jar2" ), ... )`

Reply:
I shamelessly copied an existing test with this content.
They are not useful because the `java` arg file content is displayed anyway. And I'm not really sure what I should verify with these paths.
So I rather like to delete these debug lines. WDYT ?

Reply:
After you have deleted it, we can force the build to run.